LayoutTests/mathml/non-core/scripts-subsup.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!doctype html><!-- webkit-test-runner [ CoreMathMLEnabled=false ] -->
```

Comment:
I'm not sure what the changes to these subsup and underover tests are about?

maybe  we should rewrite them to use mn and numbers instead of mi and letters, if that cause weird interaction with text-transform.